skimage/restoration/tests/test_denoise.py

```diff
@@ -153,29 +153,29 @@ def test_denoise_tv_bregman_3d():
 
 
 def test_denoise_bilateral_2d():
-    img = checkerboard_gray.copy()
+    img = checkerboard_gray.copy()[:50,:50]
     # add some random noise
     img += 0.5 * img.std() * np.random.rand(*img.shape)
     img = np.clip(img, 0, 1)
 
     out1 = restoration.denoise_bilateral(img, sigma_color=0.1,
-                                         sigma_spatial=20, multichannel=False)
```

Comment:
This will still result in a window size of 2 * 3 * 10 + 1= 61, which is rather large considering the image is only 50x50.